jobqueue_features/clusters.py

```diff
@@ -562,15 +611,21 @@ def _update_kwargs_env_extra(self, **kwargs) -> Dict[str, Any]:
 
 
 class CustomSLURMCluster(CustomClusterMixin, SLURMCluster):
-    """Custom SLURMCluster class with CustomClusterMixin for initial kwargs tweak.
-     adds client attribute to SLURMCluster class."""
+    __doc__ = f"""Custom SLURMCluster class with CustomClusterMixin for initial kwargs tweak.
+
+     Adds client attribute to SLURMCluster class.
+
+     Attributes
+     ----------
+     {custom_cluster_attributes}
+     """
```

Comment:
I'm probably showing my ignorance here but shouldn't this be
```
__doc__ = f"""...""".format{custom_cluster_attributes=custom_cluster_attributes}
```

Reply:
This is an f-string so the `.format(foo=bar)` or `.format(bar)` is unnecessary.